stdlib/public/Platform/Darwin.swift

```diff
@@ -23,3 +23,9 @@ public let M_PI_2 = Double.pi / 2
 
 @available(swift, deprecated: 3.0, message: "Please use 'Double.pi / 4' or '.pi / 4' to get the value of correct type and avoid casting.")
 public let M_PI_4 = Double.pi / 4
+
+@available(swift, deprecated: 3.0, message: "Please use '2.squareRoot()'.")
+public let M_SQRT2 = 2.squareRoot()
```

Comment:
I'm more inclined to leave the recommendation as is, unless anyone has strong objections.
BTW, is there a reason why `sqrt` is not available without any imports?